### PR TITLE
[benchmarks][dynamo] Trying CI - Set train() for TIMM models accuracy tests

### PR DESCRIPTION
**Pull request by anijain2305**
Moving to train mode for TIMM models and also raising batch size for accuracy testing.

Raising batch size seems to remove a lot of noise/instability coming from batch_norm decomposition.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire @ngimel

### CODE REVIEW

**Review by ngimel — APPROVED** [review]

**Comment by desertfire** [comment]
LGTM
